src/main.ts

```diff
@@ -53,15 +55,22 @@ export async function initRemix({
     app.whenReady(),
   ])
 
+  let lastBuildTime: number = 0;
```

Comment:
The type here can be inferred
```suggestion
  let lastBuildTime = 0;
```